src/app/app.component.html

```diff
@@ -38,30 +38,43 @@
 	<div class="usa-modal__content">
 		<div class="usa-modal__main">
 			<h2 class="usa-modal__heading" id="helpModalHeading">
-				Help
+				About & Help
 			</h2>
-			<div class="usa-prose">
-				<p id="helpModalDescription">
-					Please send feedback, questions, or issues to <a href="mailto:pmccarth@usgs.gov">pmccarth@usgs.gov</a>.
-				</p>
-			</div>
+
+			<div class="modal-body">
```

Comment:
Also include modal content from the dev branch: https://github.com/USGS-WiM/StreamStats-National/blob/dev/src/app/app.component.html#L76L96